### PR TITLE
Fix NSRangeException crash in swiftlint

### LINKED ISSUE #294: NSRangeException crash in swiftlint
This code produces crash in the very latest master source code

```swift
//
//  Experiments.swift
//  Modules
//
//  Created by Сергей Галездинов on 25.12.15.
//  Copyright © 2015 Сергей Галездинов. All rights reserved.
//

import Foundation

/**
 Test description

 - ValueOne   Значение раз
 - ValueTwo   Значение два
 - ValueThree Значение три
 - ValueFour  Значение четыре
 */
public struct TestFlags: OptionSetType {
    /// Поле, хранящее целочисленное представление флагов
    /// - seealso: `RawRepresentable`
    public let rawValue: Int

    /// Значение раз
    public static let Delivered    = TestFlags(rawValue: 0)

    /// Значение два
    public static let Queued = TestFlags(rawValue: 1 << 0)

    /// Значение три
    public static let DeletePending = TestFlags(rawValue: 1 << 1)

    /// Значение четыре
    public static let Edited  = TestFlags(rawValue: 1 << 2)

    /// Конструктор
    /// - Parameter rawValue: "Сырое" представление множества
    public init(rawValue: Int) {
        self.rawValue = rawValue
    }
}
```

produces crash:
> *** Terminating app due to uncaught exception 'NSRangeException', reason: '*** -[NSRegularExpression enumerateMatchesInString:options:range:usingBlock:]: Range or index out of bounds'

full stack trace:
http://pastie.org/10651819

### PR DESCRIPTION
**Pull request by norio-nomura**

### CODE REVIEW

**Comment by norio-nomura** [comment]
Sorry, I don't add prefix to name of branch.

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,10 @@
   [JP Simard](https://github.com/jpsim)
   [#279](https://github.com/realm/SwiftLint/issues/279)
 
+* NSRangeException crash in swiftlint.
```
Comment (jpsim) [inline CHANGELOG.md]: Same here: ```Fix an `NSRangeException` crash.  ``` (also, 2 trailing spaces)

**Comment by jpsim** [comment]
:+1: other than changelog entry :smile:

**Comment by norio-nomura** [comment]
Thank you for review!
I pushed.
:pray:

**Pull request merged**